gufe/tests/test_smallmoleculecomponent.py

```diff
@@ -228,6 +231,9 @@ def test_deserialize_roundtrip(self, toluene, phenol):
         for x, y in zip(pos1, pos2):
             assert (x == y).all()
 
+    # TODO: determine if we want to add our own serializers for e.g. JSON
+    # based on `to_dict`
```

Comment:
Can we make a decision on this in this PR? The xfail here is a signifant contributor to a drop in coverage (the fail happens early).

My vote is to not support a direct JSON serialization of these objects. I'm basing that on the opinion that any user who knows that they want a JSON dump will also be smart enough to figure out how to do `json.dumps(obj.to_dict())`. If we support it, it should be in the base class.

Reply:
I'm fine with leaving it here. I've added exclusion of the test modules to `.coveragerc`; this has reduced coverage further in this instance, but makes sense in my mind to do.